src/components/GroupedList/GroupedList.tsx

```diff
@@ -1,55 +1,64 @@
-/* @flow */
-
-import * as React from 'react';
+import React from 'react';
 import { View, StyleSheet, SectionList } from 'react-native';
 
 import Sections from './Sections';
 import { withTheme } from '../../core/theming';
 import { Headline } from '../Typography';
 
-import type { ViewStyleProp } from 'react-native/Libraries/StyleSheet/StyleSheet';
-import type { Theme } from '../../types/Theme';
+import { ViewStyle } from 'react-native';
+import { Theme } from '../../types/Theme';
 
 const alphabet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ#'.split('');
 
+type Item = {
+  key: number;
+  name: string;
+  group: string;
+};
+
 type Props = {
-  theme: Theme,
-  items: Array<any>,
-  groupBy: (item: any) => string,
-  renderItem: (data: { item: *, index: number }) => ?React.Element<*>,
+  theme: Theme;
+  items: Array<Item>;
+  groupBy: (item: Item) => string;
+  renderItem: (data: {
+    item: any;
+    index: number;
+  }) => React.ReactElement<any> | null;
```

Comment:
Same in other places

```suggestion
  }) => React.ReactNode;
```

Reply:
i am unable to get this change to work I think because the underlying section list types are all "React.ReactElement | null" 

```
Overload 1 of 2, '(props: Readonly<SectionListProps<Item>>): SectionList<Item>', gave the following error.
    Type '(data: { item: any; index: number; }) => ReactNode' is not assignable to type 'SectionListRenderItem<Item>'.
```